examples/riot/coap_federated_lf/Makefile

```diff
@@ -0,0 +1,35 @@
+REACTOR_UC_PATH ?= $(CURDIR)/../../../
+
+# The name of the LF application inside "./src" to build/run/flash etc.
+LF_MAIN ?= CoapFederatedLF
+FEDERATE ?= r1
```

Comment:
@LasseRosenow , I took the liberty to rename this from FEDERATION to FEDERATE. The entire distributed program is a federation, while each node is a federate

Reply:
Thank you :)